src/StaticBasemapTileLayer.js

```diff
@@ -16,6 +16,9 @@ import { TileLayer, setOptions } from 'leaflet';
 import { getStaticBasemapTilesUrl, fetchAttribution } from './Util';
 import { Util } from 'esri-leaflet';
```

Comment:
```suggestion
//import { Util } from 'esri-leaflet';
```
don't need this dependency anymore - hopefully Esri Leaflet Util works for this in the future though.